packages/inter-protocol/src/clientSupport.js

```diff
@@ -228,12 +228,12 @@ const makeBidOffer = (_brands, opts) => {
   });
   const { parseAmount } = opts;
   const proposal = {
-    give: { Currency: parseAmount(opts.give) },
+    give: { Bid: parseAmount(opts.give) },
```

Comment:
This change in the contract offer shape is a breaking change, right?

This new client code won't interoperate with, for example, our internal network that's running 5fcfe1a89c.